packages/turbo-repository/rust/src/lib.rs

```diff
@@ -63,4 +99,29 @@ impl Workspace {
     pub async fn find_packages(&self) -> std::result::Result<Vec<Package>, napi::Error> {
         self.packages_internal().await.map_err(|e| e.into())
     }
+
+    #[napi]
+    pub async fn package_graph(&self) -> Result<HashMap<String, Vec<String>>, Error> {
+        let packages = self.find_packages().await?;
+
+        let workspace_path = match AbsoluteSystemPath::new(self.absolute_path.as_str()) {
+            Ok(path) => path,
+            Err(e) => return Err(Error::from_reason(e.to_string())),
+        };
+
+        let map: HashMap<String, Vec<String>> = packages
+            .iter()
+            .map(|package| {
+                let deps = package.dependents(&self.graph, workspace_path);
+                let dep_names = deps
+                    .iter()
+                    .map(|p| p.relative_path.clone())
+                    .collect::<Vec<String>>();
+
+                (package.relative_path.clone(), dep_names)
+            })
```

Comment:
We can avoid a few allocations by consuming the `Vec<Package>` instead of borrowing from them
```suggestion
            .into_iter()
            .map(|package| {
                let deps = package.dependents(&self.graph, workspace_path);
                let dep_names = deps
                    .into_iter()
                    .map(|p| p.relative_path)
                    .collect::<Vec<String>>();

                (package.relative_path, dep_names)
            })
```